src/ADAL.PCL/ADAL.PCL.csproj

```diff
@@ -84,6 +84,18 @@
     <Compile Include="Base64UrlEncoder.cs" />
     <Compile Include="CacheQueryData.cs" />
     <Compile Include="CallState.cs" />
+    <Compile Include="IDispatcher.cs" />
+    <Compile Include="Internal-Telemetry\AggregatedDispatcher.cs" />
```

Comment:
It does not allow "\"

---
In reply to: [74538128](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r74538128) [](ancestors = 74538128)